docs/usage.md

```diff
@@ -93,6 +93,12 @@ genome: 'GRCh37'
 
 You can also generate such `YAML`/`JSON` files via [nf-core/launch](https://nf-co.re/launch).
 
+Optionally, the `sample_size` parameter allows you to subset a random number of reads to be analysed.
+
+```bash
+nextflow run nf-core/seqinspector --input ./samplesheet.csv --outdir ./results --sample_size 90 -profile docker
```

Comment:
Ok, yup, makes sense. I chose that number because was the same for the test, but I have changed it to 1 million reads. I guess it also depends on the application, 16S may require less reads than metagenomics. Please, let me know if you have another number in mind.

Reply:
Just out of curiosity: Have you tested what happens, if the number of reads to subsample is larger than the input file size? Does the tool just pass through the whole file or crash?